Demisto-sdk release 1.38.7 (#5010)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## 1.38.7 (2025-07-20)
### Feature

> * Add support for supportedModules field in integration, command, and argument. [#4996](https://github.com/demisto/demisto-sdk/pull/4996)

Reviewer: ```suggestion
* Added support for `supportedModules` field in integrations, commands, and command arguments. [#4996](https://github.com/demisto/demisto-sdk/pull/4996)
```

* Added support for PLATFORM marketplace and multi-marketplace tags (e.g., <~XSIAM,PLATFORM>) in release notes. [#4982](https://github.com/demisto/demisto-sdk/pull/4982)

### Fix

> * fixed a syntax issue when printing feed parameters related errors. [#4995](https://github.com/demisto/demisto-sdk/pull/4995)

Reviewer: ```suggestion
* Fixed a syntax issue when printing feed parameters related errors. [#4995](https://github.com/demisto/demisto-sdk/pull/4995)
```

* Addressed `pkg_resources` deprecation warning by migrating to `importlib.metadata`. [#5002](https://github.com/demisto/demisto-sdk/pull/5002)

> * Remove redundant SupportedModulesValues ENUM and import the existing one. [#5000](https://github.com/demisto/demisto-sdk/pull/5000)

Reviewer: ```suggestion
* Removed redundant `SupportedModulesValues` enum. [#5000](https://github.com/demisto/demisto-sdk/pull/5000)
```

> * Fixed SupportedModulesValues to have agentix in the list. [#4996](https://github.com/demisto/demisto-sdk/pull/4996)

Reviewer: ```suggestion
* Updated `SupportedModulesValues` to support agentix. [#4996](https://github.com/demisto/demisto-sdk/pull/4996)
```

### Internal

> * Fix handling of playbooks in GR110 validation. [#5008](https://github.com/demisto/demisto-sdk/pull/5008)

Reviewer: ```suggestion
* Fixed handling of playbooks in *GR110* validation. [#5008](https://github.com/demisto/demisto-sdk/pull/5008)
```

> * Added the IN164 validation. The validation ensures that a required param in an existing integration has a default value. [#5006](https://github.com/demisto/demisto-sdk/pull/5006)

Reviewer: ```suggestion
* Added the *IN164* validation. The validation ensures that a required param in an existing integration has a default value. [#5006](https://github.com/demisto/demisto-sdk/pull/5006)
```
